spec/services/atmosphere/billing_service_spec.rb

```diff
@@ -236,11 +311,11 @@
       expect appl3b.amount_billed = cost_unit*1
       expect expiring_appl.amount_billed = cost_unit
 
-      expect(appl1.billing_state).to eq "prepaid"
-      expect(appl2.billing_state).to eq "prepaid"
-      expect(appl3a.billing_state).to eq "prepaid"
-      expect(appl3b.billing_state).to eq "prepaid"
-      expect(expiring_appl.billing_state).to eq "expired"
+      expect(appl1.deployments.first.billing_state).to eq "prepaid"
+      expect(appl2.deployments.first.billing_state).to eq "prepaid"
+      expect(appl3a.deployments.first.billing_state).to eq "prepaid"
+      expect(appl3b.deployments.first.billing_state).to eq "prepaid"
+      expect(expiring_appl.deployments.first.billing_state).to eq "expired"
```

Comment:
I see 5 specs here - please split it. As @tbartynski mentioned one spec should test only one thing.